lnrpc/rpc.proto

```diff
@@ -2371,11 +2371,11 @@ message Payment {
     /// Deprecated, use value_sat or value_msat.
     int64 value = 2 [json_name = "value", deprecated = true];
 
-    /// The date of this payment
-    int64 creation_date = 3 [json_name = "creation_date"];
+    /// Deprecated, use creation_time_ns
+    int64 creation_date = 3 [json_name = "creation_date", deprecated = true];
 
-    /// The path this payment took
-    repeated string path = 4 [ json_name = "path" ];
+    /// Deprecated, use path on individual htlcs
```

Comment:
should we deprecate if the alternative is experimental?

Reply:
touché, undeprecating